### DISCUSSION TITLE
브랜치 이름을 컨벤션에 맞게 수정해주세요👀

### DISCUSSION THREAD

**Discussion by compuTasha** [DeveloperAcademy-POSTECH/MacC-GetARock#54] [오류 제보하기]
<img width="405" alt="image" src="https://user-images.githubusercontent.com/51108729/202909378-06386b5b-016b-48bb-a896-bcf3b597747d.png">

### 브랜치 네이밍 수정

[Git Branch 전략 위키](https://github.com/DeveloperAcademy-POSTECH/MacC-GetARock/wiki/1%EF%B8%8F⃣%C2%A0Git-branch-전략)를 참고하면 작업중인 브랜치들은 `feature/{issue-number}-{feature-name}` 의 형식을 따라야 합니다..!

아마 `feature` 라는 이름 때문에 저 자리에 [타입]이 들어간다고 생각하실 수 있지만
[우린 Git-flow를 사용하고 있어요 | 우아한형제들 기술블로그](https://techblog.woowahan.com/2553/)를 참고하시면 
`feature`는 '기능을 개발하는 브랜치'로써 `develop`, `master` 와 같이 그 이름 그대로 써주시면 된답니다🤔

즉 `design/37-gathering-information-page-layout`과 같은 브랜치 네이밍을
`feature/37-gathering-information-page-layout` 과 같이 변경하면 됩니다!

<br>

### 수정 방법

깃크라켄, fork 와 같은 툴을 사용하신다면 브랜치 우클릭을 통해 빠르게 이름을 변경하실 수 있습니다.
터미널을 통해 변경하신다면 [[Git] Branch 이름 변경하기](https://llshl.tistory.com/m/68)를 참고해주세요!

<br>

### ☝️ 추가로...
**PR 후 머지가 된 브랜치는 삭제해주세요!**  
<br>

@skycat0212 @EuniceNam @GODNOEL @siro96-01 @Somin-DS @taehyeonk @unuhqueen

**Comment by skycat0212** [comment]
제가 사용했던 머지 완료된 브랜치 삭제하겠습니다!

**Comment by MARKGRAYSIRO** [comment]
오호!! 진짜 몰랐어요 ㅠㅠㅠㅠ 수정해서 반영하겠습니다!! 갓 로라 땡큐...!!
